packages/scrapers-v2/src/urls/types.ts

```diff
@@ -0,0 +1,21 @@
+export enum College {
+  ARTS_MEDIA_DESIGN = "arts-media-design",
+  BUSINESS = "business",
+  KHOURY = "computer-information-science",
+  ENGINEERING = "engineering",
+  HEALTH_SCIENCES = "health-sciences",
+  SCIENCE = "science",
+  SOCIAL_SCIENCES_HUMANITIES = "social-sciences-humanities",
+}
+
+export type CatalogHierarchy = {
```

Comment:
Add comments: reflects structure of catalog hierarchy, with each sub level represented by an onject. "Leaves" are the full URLs to each entry.

Reply:
agreed! a diagram-like comment like the one in the first stage pr would be super nice